prog/dftb+/lib_dftb/energytypes.F90

```diff
@@ -8,13 +8,13 @@
 #:include 'common.fypp'
 
 !> Module to wrap around the different energy components in the DFTB total energy expression
-module dftbp_energies
+module dftbp_energyTypes
```

Comment:
The naming convention for modules prefers snakecase here, please avoid capital letters in module names.